README.md

```diff
@@ -48,6 +48,9 @@ Start-BitsTransfer -Source https://git.io/rustlings-win -Destination $env:TMP/in
 
 To install Rustlings. Same as on MacOS/Linux, you will have access to the `rustlings` command after it.
 
+## Browser:
+[![Run on Repl.it](https://repl.it/badge/github/rust-lang/rustlings)](https://repl.it/github/rust-lang/rustlings)
```

Comment:
I'd prefer a small piece of text instead of a small and hard-to-parse badge here.

Reply:
@fmoko, sounds good, fixed!